scripts/ErrorReporter/retrieve_recovery_files.py

```diff
@@ -0,0 +1,45 @@
+from mantid.kernel import ConfigService
+import os
+import shutil
+import hashlib
+import datetime
+
+
+class RetrieveRecoveryFiles(object):
```

Comment:
Why does this need to be a class at all? A module of free functions would be much cleaner.

Reply:
Given all the methods are static it doesn't need to be a class at all. You're correct a module of free functions would be cleaner. I'll update.